app-grid/app-grid-style.html

```diff
@@ -0,0 +1,59 @@
+<!--
+@license
+Copyright (c) 2016 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+
+<link rel="import" href="../../polymer/polymer.html">
+<link rel="import" href="../../iron-flex-layout/iron-flex-layout.html">
+
+<dom-module id="app-grid-style">
+  <template>
+    <style>
+
+      :host {
+        --grid-expandible-item: {
+          flex-basis: calc((100% - (var(--gutter, 0px) * (var(--columns, 1) - var(--item-columns, 1)))) / var(--columns, 1) * var(--item-columns, 1)) !important;
+          max-width: calc((100% - (var(--gutter, 0px) * (var(--columns, 1) - var(--item-columns, 1)))) / var(--columns, 1) * var(--item-columns, 1))  !important;
+        };
+      }
+
+      .app-grid {
+        @apply(--layout-horizontal);
+        @apply(--layout-wrap);
+        margin-top: var(--gutter, 0px);
+        margin-left: var(--gutter, 0px);
+      }
+
+      .app-grid > *:not(template) {
+        display: block;
+        position: relative;
```

Comment:
`position: relative;` is needed only for `has-aspect-ratio`. Maybe we should only add it when `.app-grid[has-aspect-ratio]`